### PR TITLE
OSSM-3695: 2.3.3 Z Stream Release Notes, Known Issues and Bug Fixes for OSSM 

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[DOC] 2.3.3 Z Stream Release Notes, Known Issues and Bug Fixes for OSSM

Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OSSM-3695

Link to docs preview:

New features: https://58341--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-version-2-3-3

Fix issues: https://58341--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#ossm-rn-fixed-issues-ossm_ossm-release-notes

Please review the following issues:

[OSSM-3644](https://issues.redhat.com//browse/OSSM-3644)
[OSSM-3595](https://issues.redhat.com//browse/OSSM-3595)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Federation egress-gateway gets wrong update of network gateway endpoints, causing extra endpoint entries. The federation egress-gateway has been updated so it gets the correct update of network gateway endpoints.
```
Comment (bmangoen) [inline modules/ossm-rn-fixed-issues.adoc:22]: We might change `The federation egress-gateway has been updated so it gets the correct update of network gateway endpoints.` to something like this `This has been fixed on the server side so it gets the correct update of network gateway endpoints`. WDYT ?

Comment (gwynnemonahan) [reply modules/ossm-rn-fixed-issues.adoc:22]: Doc style prefers not to use phrases like "It is fixed" or "The issue is resolved" or "The error no longer presents" so how about ... `This has been updated on the server side so it gets the correct update of network gateway endpoints.`

Comment (gwynnemonahan) [reply modules/ossm-rn-fixed-issues.adoc]: Per convo via Slack, updated to say `This has been updated on the server side so it gets the correct network gateway endpoints.`

**Comment by pbajjuri20** [comment]
SMCP V2.1: Kiali : 1.36.16

@gwynnemonahan : We need the update the Kiali container version for the 2.1 SMCP.

**Comment by gwynnemonahan** [comment]
> 

Updated.

**Comment by gwynnemonahan** [comment]
@pbajjuri20 the `SMCP V2.1: Kiali : 1.36.16 ` update has been made.

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-rn-fixed-issues.adoc [hunk 2]
```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Federation egress-gateway gets the wrong update of network gateway endpoints, causing extra endpoint entries. This has been updated on the server side so it gets the correct network gateway endpoints.
```
Comment (bburt-rh) [inline modules/ossm-rn-fixed-issues.adoc:22]: - The bug fix description needs to be rewritten using the guidelines and template in the SSG: "Use past tense for the problem and present tense for the solution". See: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes
- I suggest rewording the second sentence to make specific what "this" and "it" refer to. It's not clear right now. "This" what? "This issue"? What precisely "gets the correct network gateway endpoints"?

File: modules/ossm-rn-fixed-issues.adoc [hunk 3]
```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
...
+
+* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Sometimes, `istio-cni` fails on RHEL9 because of SELinux where `iptables-restore` was not allowed to open files in `/tmp`. SELinux now passes `iptables-restore` via `stdin`.
```
Comment (bburt-rh) [inline modules/ossm-rn-fixed-issues.adoc:24]: - The bug fix description needs to be rewritten using the guidelines and template in the SSG: "Use past tense for the problem and present tense for the solution". See: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes
- Also, I think that `RHEL9` is not correct. As a product name, it needs to use an attribute -- either `{op-system-base}` or `{op-system-base-full}`

File: modules/ossm-rn-fixed-issues.adoc [hunk 4]
```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Previously, the federation egress-gateway received the wrong update of network gateway endpoints, causing extra endpoint entries. Now, the federation-egress gateway has been updated on the server side so it receives the correct network gateway endpoints.
+
+* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Previously, `istio-cni` sometimes failed on {op-system-base} because SELinux did not allow `iptables-restore` to open files in `/tmp` directory. Now, SELinux passes `iptables-restore` via `stdin` instead of via a file.
```
Comment (bburt-rh) [inline modules/ossm-rn-fixed-issues.adoc:24]:
```suggestion
* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Previously, `istio-cni` sometimes failed on {op-system-base} because SELinux did not allow `iptables-restore` to open files in the `/tmp` directory. Now, SELinux passes `iptables-restore` via `stdin` instead of via a file.
```
- Sorry, I missed it before, my bad. But all of the backticked literals in this sentence need referents. E.g.  what is `istio-cni`? a tool? a utility? an object? a component?

Comment (rolfedh) [reply modules/ossm-rn-fixed-issues.adoc]: @gwynnemonahan Please mark conversations like this one as resolved. Thanks.

**Comment by bburt-rh** [comment]
lgtm
